commit a67464b06174f742de2afc4de29b92ef734a5bdf (refs/changes/36/1536/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-02-17T17:09:02-08:00 (2 years, 8 months ago)
    
    joyent/node-triton#176 triton `--act-as` option was broken by joyent/node-triton#156

diff --git a/lib/config.js b/lib/config.js
index 7ea03dd..f109662 100644
--- a/lib/config.js
+++ b/lib/config.js
@@ -61,7 +61,8 @@ var PROFILE_FIELDS = {
     account: true,
     keyId: true,
     insecure: true,
-    user: true
+    user: true,
+    actAsAccount: true
 };
 
 
@@ -232,6 +233,7 @@ function validateProfile(profile, profilePath) {
         assert.optionalString(profile.user,
             profile.name === 'env' ? 'TRITON_USER or SDC_USER'
                 : 'profile.user');
+        assert.optionalString(profile.actAsAccount, 'profile.actAsAccount');
     } catch (err) {
         var msg = format('invalid %sprofile%s: %s',
             profile.name ? '"' + profile.name + '" ' : '',
